devtools/lightning-inspect.js

```diff
@@ -268,9 +268,13 @@ window.attachInspector = function({Application, Element, ElementCore, Stage, Com
             c = c._element;
         }
         if (v == dv) {
-            c.dhtmlRemoveAttribute(n);
+            if (c.dhtmlRemoveAttribute) {
```

Comment:
Same logic as with parent.dhtml; in some of my testing the code was getting to line 271 with an error that c object didn't have a .dhtmlRemoveAttribute function; adding this second if-check fixes that.